index.html

```diff
@@ -222,7 +229,10 @@ <h1 class="section-title">
             <button type="submit" class="form-submit-btn">Get in touch</button>
           </form>
         </div>
-        <img class="illus-1" src="images/illustration-contact-form.png" alt="illustration">
+        <img class="illus-mobile illus-1" src="images/illustration-contact-form.png" alt="illustration">
+        <img class="illus-desktop illus-1" src="images/illustration-1-contact-form-desktop.png" alt="illustration">
+        <img class="illus-desktop illus-2" src="images/illustration-2-contact-form-desktop.png" alt="illustration">
+        <img class="illus-desktop illus-3" src="images/illustration-3-contact-form-desktop.png" alt="illustration">
       </section>
     </main>
     <footer>
```

Comment:
You have done excellent work on the desktop version and in my opinion this project should be approved straight away 🔥. However, just to make your project look exactly like the template design, I'd like to point out this difference I spotted between your design and the Figma design: 

- [ ] In your footer, you placed a p tag containing an email (line 239) which is not consistent with Figma design. While this may look good, I suggest that you develop the habit of sticking as much as possible to design templates as this is a key trait in a professional working environment.
- [ ] In the design template, above the social icons in the footer, there is a horizontal line but I did not see it in your design. I suggest that you place an ```<hr>``` tag just above the ```ul``` tag on line 240 so as to add the horizontal line and make your design look just like the Figma template.

Your design: 
![image](https://github.com/thasipablo/Personal-Portfolio/assets/107795045/0eb169f3-0b9f-4016-9d23-d959f0258087)

Design template: 
![image](https://github.com/thasipablo/Personal-Portfolio/assets/107795045/1f9603b3-c877-44c1-95ab-e33480f71cc0)